docs/program/apis/_index.md

```diff
@@ -342,400 +342,16 @@ In addition to the [board API](#board), the [board component](/components/board/
 
 {{< readfile "/static/include/components/apis/digitalinterrupt.md" >}}
 
-## Robot API
+## Robot Management APIs
```

Comment:
This seems perfect

Reply:
Thanks! Wait what do you mean about being done editing?